### DISCUSSION TITLE
gatsby-source-strapi": "^3.2.0", and "gatsby": "^5.9.1"

### DISCUSSION THREAD

**Discussion by gosrinin** [gatsby-uc/plugins#429] [Q&A] [answered]
Error: The plugin "gatsby-source-strapi" created a node of a type owned by another plugin.
        The node type "File" is owned by "gatsby-source-filesystem".
        If you copy and pasted code from elsewhere, you'll need to pick a new type name
        for your new node(s).
        The plugin creating the node:

i am gettign this error unable to move forward can anyone help me out

attached the gatsby-config.js file for your reference.

`require("dotenv").config({
  path: `.env.${process.env.NODE_ENV}`,
})

module.exports = {
  plugins: [
    {
      resolve: "gatsby-source-strapi",
      options: {
        apiURL: process.env.STRAPI_API_URL || "http://localhost:1337",
        accessToken: process.env.STRAPI_TOKEN,
        collectionTypes: [
          {
            singularName: "blog",
          },
          {
            singularName: "author",
          },
          {
            singularName: "category",
          },
        ],
        singleTypes: [ ],
      },
    },
    {
      resolve: 'gatsby-source-filesystem',
    },
    "gatsby-plugin-image",
    "gatsby-plugin-sharp",
    "gatsby-transformer-sharp",
    "gatsby-transformer-remark",
  ],
}`

**Comment by moonmeister** [comment, marked as answer]
There is an open issue for this error. Happy to take a PR for a fix.
